### DISCUSSION TITLE
Pi Juice set up

### DISCUSSION THREAD

**Discussion by gencosystems** [PiSupply/PiJuice#1073] [General]
Can somebody guide me as to how to set the pi juice up in the following way please....

When power applied the raspberry pi would boot up.

When power removed the pi juice would initiate a soft shut down so as to protect the SD card.

Many thanks
Graham.

**Comment by tvoverbeek** [comment]
When power applied the raspberry pi would boot up.
   Set wakeup on charge enabled with trigger level 0% (System task tab)

When power removed the pi juice would initiate a soft shut down so as to protect the SD card.
   use the "no power" event to trigger SYS_FUNC_HALT_POW_OFF (System events tab)

**Reply by gencosystems** [reply]
Thank you very much that worked very well.
Many thanks
Graham

**Reply by gencosystems** [reply]
Hello,
What's the significance of using the pi Juice usb power input rather than the pi usb power input?

Would using the pi juice power input change the set up you have previously mentioned?

Many thanks
Graham.
